hbase-http/src/main/java/org/apache/hadoop/hbase/http/prometheus/PrometheusHadoopServlet.java

```diff
@@ -41,7 +41,10 @@ public class PrometheusHadoopServlet extends HttpServlet {
 
   @Override
   protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
-    writeMetrics(resp.getWriter());
+    String tmpStr = req.getParameter("description");
+    boolean description = tmpStr != null && tmpStr.equals("true");
+
+    writeMetrics(resp.getWriter(), description);
```

Comment:
This could be compacted like this:
`writeMetrics(resp.getWriter(), "true".equals(req.getParameter("description"))); `

Reply:
I totally agree, should have noticed that. Thank you!